testsuite/layers-expansion/pom.xml

```diff
@@ -760,6 +760,7 @@
                                                 <layer>messaging-activemq</layer>
                                                 <layer>micrometer</layer>
                                                 <layer>mod_cluster</layer>
+                                                <layer>mvc-krazo</layer>
```

Comment:
Perhaps add a comment that says that being preview, nothing is provisioned although present. I understand that this comment would have to be removed when promoting this subsystem.

Reply:
This one is a mistake. It should not be in this execution until it will actually be provisioned. Adding it here would be part of promoting the feature to community.

But commenting it out with a comment is better than deleting it. :)